packages/react/src/index.ts

```diff
@@ -2,7 +2,7 @@ export * from '@sentry/browser';
 
 export { init } from './sdk';
 export { Profiler, withProfiler, useProfiler } from './profiler';
-export { ErrorBoundary, withErrorBoundary } from './errorboundary';
+export { ErrorBoundary, FallbackRender, withErrorBoundary } from './errorboundary';
```

Comment:
We'll have to use type only exports here as we've turned on [`isolatedModules`](https://www.typescriptlang.org/tsconfig#isolatedModules) - as we don't use TSC to transpile our TypeScript, we use [Sucrase](https://github.com/alangpierce/sucrase)

```suggestion
export type { FallbackRender } from './errorboundary';
export { ErrorBoundary, withErrorBoundary } from './errorboundary';
```

Reply:
Oh cool, I had no idea of that feature of tsc.